ehrql/tables/emis.py

```diff
@@ -38,7 +38,7 @@ class patients(PatientFrame):
     There is generally a lag between the death being recorded in ONS data and it
     appearing in the primary care record, but the coverage or recorded death is almost
     complete and the date of death is usually reliable when it appears. There is
-    also a lag in ONS death recording (see [`ons_deaths`](/reference/schemas/core/#ons_deaths)
+    also a lag in ONS death recording (see [`ons_deaths`](core.md#ons_deaths)
```

Comment:
thought: I've fixed up this link to be a relative link with the `.md` suffix. (The link is actually currently broken on the OpenSAFELY documentation site, because it's an absolute link, but the OpenSAFELY documentation site path is actually `/ehrql/references/schemas/core`.)

However, there's one of two things wrong here, even in the fixed link:

1. either, the core — and maybe the EMIS table — should warn about the ONS death recording lag; they don't, only the TPP entry includes this warning
2. or, the link should go to the TPP table

I'm guessing 1 is correct and not 2.

Reply:
I think this comment should be broken out into a new issue. I also think 1 is correct and not 2. @LFISHER7 added the information about the recording lag IIRC, so he may be able to answer the question definitively.

Reply:
Yes, I agree, the warning isn't backend specific so should be surfaced for all the tables

Reply:
Because this is incidental, and so I can merge this PR to clean up my work on ehrQL before the holiday period, I've left this link as is and opened [#1853](https://github.com/opensafely-core/ehrql/issues/1853).